### ISSUE TITLE
Changing a users forgotten pasword / reseting it in the webinterface.

### ISSUE THREAD

**Issue by x0rp01s0n** [MrLetsplay2003/ShittyAuthServer#14]
As the title suggests, it would be very helpful to have the feature to set or reset a password for a specific user in the administration menu.

**Comment by MrLetsplay2003** [comment]
Yes, that is something I had wanted to implement as well. There is an API endpoint for it in the admin API, however you can only use it on the (unfinished) new UI, not the default one. I'll probably add it to the default UI as well though :+1:

**Comment by MrLetsplay2003** [comment]
I just added an separate admin page in 28eb125 with this functionality. The "Change WIAPI password" feature is currently also included there, but I will move it to the main WebinterfaceAPI repo at some point.
